lib/jsdom/living/events/MouseEvent-impl.js

```diff
@@ -7,6 +7,19 @@ const UIEventImpl = require("./UIEvent-impl").implementation;
 const MouseEventInit = require("../generated/MouseEventInit");
 
 class MouseEventImpl extends UIEventImpl {
+  get x() {
+    return this.clientX;
+  }
+  get y() {
+    return this.clientY;
+  }
+  get offsetX() {
```

Comment:
Per spec these should be the same as pageX/pageY. A test, or enabling of an existing test, would be good.

Reply:
Done